client/app/scripts/superdesk-authoring/styles/themes.less

```diff
@@ -51,6 +50,279 @@ body, html {
 	.box-sizing(border-box);
 	max-width: 900px;
 
+        header {
```

Comment:
not sure it should be part of themes file, maybe you can put this to `authoring.less` ?